### ISSUE TITLE
Components only support

### ISSUE THREAD

**Issue by saendu** [TroyAlford/react-jsx-parser#53]
Hi Troy

We are trying to use the jsx-parser to enable end users interactively create their own controls 'on the fly' in a pseudo-language kind of way. What we are trying to avoid is them using plain HTML.

E.g. we want them to be able to use the parser as follows:

Admin input field:
```
<UserComponent>
    <PersonComponent img={user.img} department={user.dep} ... />
    <Button />
</UserComponent>
```

This would then render a user component based on the provided input above. 
What we would not like to see is something like this:
```
<UserComponent>
  <div>
    <h1>SUPER USER</h1>
    <img src="user.jpg" />
    <a href='..'>CLICKME</a>
  </div>
</UserComponent>
```
At the moment we could ban specific tags, but what we want to achieve is to ban all HTML tags.
For this I would like to see a prop that we can set to only allow using components, like so:
```
<JsxParser
      bindings=...
      jsx=...
      components={{ InjectableComponent, UserComponent, PersonComponent, Button }}
      componentsOnly={true}
/>
```

Within the parseElement function you could then check for something like this:
```
if(this.props.componentsOnly && !(name in components)) return undefined; 
```

Let me know what you think. I can provide a pull request if you want. 
-Sandro

**Comment by TroyAlford** [comment]
Closed by #54 - I'll publish it shortly and release the new version.

**Comment by TroyAlford** [comment]
Published just now as 1.3.2!

**Comment by saendu** [comment]
Great thanks! Since you accepted my enhancement request we should maybe upgrade the readme too.
